tools/ColorTool/ColorTool/ColorTable.cs

```diff
@@ -0,0 +1,207 @@
+﻿//
+// Copyright (C) Microsoft.  All rights reserved.
+// Licensed under the terms described in the LICENSE file in the root of this project.
+//
+
+using System;
+using System.Collections.Generic;
+
+namespace ColorTool
+{
+    /// <summary>
+    /// Displays the color table that demonstrates the current colorscheme.
+    /// </summary>
+    class ColorTable
+    {
+        const int DARK_BLACK = 0;
+        const int DARK_BLUE = 1;
+        const int DARK_GREEN = 2;
+        const int DARK_CYAN = 3;
+        const int DARK_RED = 4;
+        const int DARK_MAGENTA = 5;
+        const int DARK_YELLOW = 6;
+        const int DARK_WHITE = 7;
+        const int BRIGHT_BLACK = 8;
+        const int BRIGHT_BLUE = 9;
+        const int BRIGHT_GREEN = 10;
+        const int BRIGHT_CYAN = 11;
+        const int BRIGHT_RED = 12;
+        const int BRIGHT_MAGENTA = 13;
+        const int BRIGHT_YELLOW = 14;
+        const int BRIGHT_WHITE = 15;
+
+        // This is the order of colors when output by the table.
+        private static readonly IReadOnlyList<int> outputFgs = new[]
```

Comment:
>outputFgs [](start = 51, length = 9)

maybe make these readonly lists have consistent naming. Only since I poked you on that in other comments and you have been doing a great job at cleaning up odds and ends like this already across these iterations (like fixing cases on methods). Won't fix if you don't want to bother. It's OK.